torchao/swizzle/swizzle_ops.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD 3-Clause license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import Any, Dict, Tuple
+
+import torch
+
+import torchao.ops
+from torchao.swizzle.swizzle_tensor import SwizzleTensor
+
+aten = torch.ops.aten
+SWIZZLE_OPS_TABLE: Dict[Any, Any] = {}
```

Comment:
Given how you use this, won't you be able to use the builtin version of this: https://pytorch.org/docs/stable/library.html#torch.library.register_torch_dispatch